apps/api/src/app/events/usecases/index.ts

```diff
@@ -1,17 +1,3 @@
-import {
-  DigestFilterSteps,
-  DigestFilterStepsBackoff,
-  DigestFilterStepsRegular,
-  DigestFilterStepsTimed,
-  AddJob,
-  AddDelayJob,
-  AddDigestJob,
-  TriggerEvent,
-  ProcessSubscriber,
-  CreateNotificationJobs,
-  StoreSubscriberJobs,
-} from '@novu/application-generic';
```

Comment:
Loading dependencies in a module here and then as USE_CASES satisfied the validation but for the automatic dependency injection was contributing to unexpected behaviour. A serious head pain. 😕